boards/adafruit/feather_esp32s2/doc/index.rst

```diff
@@ -0,0 +1,541 @@
+         :board: adafruit_feather@B/esp32s2
+         :goals: flash
+
+   .. group-tab:: adafruit_feather@C/esp32s2
+
+      .. zephyr-app-commands::
+         :zephyr-app: samples/hello_world
+         :board: adafruit_feather@C/esp32s2
+         :goals: flash
+
+   .. group-tab:: adafruit_feather_tft/esp32s2
+
+      .. zephyr-app-commands::
+         :zephyr-app: samples/hello_world
+         :board: adafruit_feather_tft/esp32s2
+         :goals: flash
+
+   .. group-tab:: adafruit_feather_tft_reverse/esp32s2
+
+      .. zephyr-app-commands::
+         :zephyr-app: samples/hello_world
+         :board: adafruit_feather_tft_reverse/esp32s2
+         :goals: flash
+
+After the flashing you will receive most likely this Error:
+
+.. code-block:: console
+
+   WARNING: ESP32-S2FNR2 (revision v0.0) chip was placed into download mode using GPIO0.
+   esptool.py can not exit the download mode over USB. To run the app, reset the chip manually.
+   To suppress this note, set --after option to 'no_reset'.
+   FATAL ERROR: command exited with status 1: ...
+
+As stated in the Warning-Message ``esptool`` can't reset the board by itself and this message
+can be ignored and the board needs to be reseted via the Reset-Button manually.
+
+Open the serial monitor using the following command:
+
+.. code-block:: shell
+
+   west espressif monitor
+
+After the board has been manually reseted and booted, you should see the following
+message in the monitor:
+
+.. code-block:: console
+
+   ***** Booting Zephyr OS vx.x.x-xxx-gxxxxxxxxxxxx *****
+   Hello World! adafruit_feather/esp32s2/...
+
+Debugging
+*********
+
+ESP32-S2 support on OpenOCD is available at `OpenOCD`_.
+
+ESP32-S2 has a built-in JTAG circuitry and can be debugged without any
+additional chip. Only an USB cable connected to the D+/D- pins is necessary.
+
+Further documentation can be obtained from the SoC vendor
+in `JTAG debugging for ESP32-S2`_.
+
+You can debug an application in the usual way. Here is an example for
+the :zephyr:code-sample:`hello_world` application.
+
+.. tabs::
+
+   .. group-tab:: adafruit_feather@B/esp32s2
+
+      .. zephyr-app-commands::
+         :zephyr-app: samples/hello_world
+         :board: adafruit_feather@B/esp32s2
+         :goals: debug
+
+   .. group-tab:: adafruit_feather@C/esp32s2
+
+      .. zephyr-app-commands::
+         :zephyr-app: samples/hello_world
+         :board: adafruit_feather@C/esp32s2
+         :goals: debug
+
+   .. group-tab:: adafruit_feather_tft/esp32s2
+
+      .. zephyr-app-commands::
+         :zephyr-app: samples/hello_world
+         :board: adafruit_feather_tft/esp32s2
+         :goals: debug
+
+   .. group-tab:: adafruit_feather_tft_reverse/esp32s2
+
+      .. zephyr-app-commands::
+         :zephyr-app: samples/hello_world
+         :board: adafruit_feather_tft_reverse/esp32s2
+         :goals: debug
+
+
+Testing the On-Board-LED
+************************
+
+There is a sample that allows to test the LED on the board are working
```

Comment:
Would you re-phrase the this (and the others) to something like:
`There is a sample that allows you to test whether the LED on the board is working properly with Zephyr:`